ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/snmp-settings.j2

```diff
@@ -7,6 +7,26 @@ snmp-server contact {{ snmp_server.contact }}
 {%     if snmp_server.location is arista.avd.defined %}
 snmp-server location {{ snmp_server.location }}
 {%     endif %}
+{%     if snmp_server.communities is arista.avd.defined %}
+{%         for community in snmp_server.communities | arista.avd.natural_sort %}
+{%             set communities_cli = "snmp-server community " ~ community %}
+{%             if snmp_server.communities[community].view is arista.avd.defined %}
+{%                 set communities_cli = communities_cli ~ " view " ~ snmp_server.communities[community].view %}
+{%             endif %}
+{%             if snmp_server.communities[community].access is arista.avd.defined %}
+{%                 set communities_cli = communities_cli ~ " " ~ snmp_server.communities[community].access %}
+{%             else %}
+{%                 set communities_cli = communities_cli ~ " ro" %}
+{%             endif %}
+{%             if snmp_server.communities[community].access_list_ipv6 is arista.avd.defined %}
+{%                 set communities_cli = communities_cli ~ " ipv6 " ~ snmp_server.communities[community].access_list_ipv6.name %}
+{%             endif %}
+{%             if snmp_server.communities[community].access_list_ipv4 is arista.avd.defined %}
+{%                 set communities_cli = communities_cli ~ " " ~ snmp_server.communities[community].access_list_ipv4.name %}
+{%             endif %}
+{{ communities_cli }}
+{%         endfor %}
+{%     endif %}
```

Comment:
This should be rendered between "view" and "group" commands.
Tested on EOS 4.23.6M:
snmp-server view VW-WRITE iso included
snmp-server community private rw
snmp-server community public ro
snmp-server community snmp-test-comm ro onur
snmp-server group GRP-READ-ONLY v3 priv read v3read